tests/storage/export.go

```diff
@@ -112,6 +112,7 @@ var _ = SIGDescribe("Export", func() {
 	var err error
 	var token *k8sv1.Secret
 	var virtClient kubecli.KubevirtClient
+	var nonRootUser = int64(107)
```

Comment:
Can this be a const? And could it have a more descriptive name? I supposed there is a reason it is UID 107 and not something else.

Reply:
consts don't have addresses :/

Reply:
And what about the naming?

Reply:
Sure, done